widgets/nuxeo-uploader-behavior.html

```diff
@@ -124,10 +124,12 @@
      * @return {Boolean} true if the current file is accepted by the current provider, false otherwise
      * */
     DefaultUploadProvider.prototype.accepts = function (file) {
-      const mimeType = ((file.type !== '') ? file.type.match(/^[^/]*\//)[0] : null);
-      const fileType = file.name.match(/\.[^.]*$/)[0];
-
-      return !(this.accept && !(this.accept.indexOf(mimeType) > -1 || this.accept.indexOf(fileType) > -1));
+      if (!this.accept) {
+        return false;
```

Comment:
if `this.accept` is empty then we should accept any type of file and therefore return `true` !